src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -206,6 +206,17 @@ public override bool SendChunked
         }
 #endif
 
+        /// <summary>
+        /// Returns the underlying <see cref="HttpClientHandler"/>.
+        /// </summary>
+        public HttpClientHandler HttpClientHandler

```

Comment:
Having conversation offline for privacy, but adding it here so we don't lose track: these types implement `IDisposable`. We probably should not be exposing them.